services/api-server/src/simcore_service_api_server/main.py

```diff
@@ -1,8 +1,7 @@
 """Main application to be deployed in for example uvicorn.
 """
 from fastapi import FastAPI
-
-from .core.application import init_app
+from simcore_service_api_server.core.application import init_app
```

Comment:
```suggestion
from .core.application import init_app
```

Reply:
@giancarloromeo As I mentioned in the description. This is a bug. This is a special file that needs absolute path (is the only one) since it is used in the CLI